docs/v2/integrations/advanced-custom-fields.md

```diff
@@ -171,6 +171,29 @@ When you run `meta` on an outer ACF field, everything inside is ready to be trav
 {% endfor %}
 ```
 
+### Split into small chunks flexible content fields
+
+We can break flexible content fields into small blocks thanks to `acf_fc_layout` value. This way we have more flexibility and reusability. It could be a way to build a landing page for instance.
+
+Assuming blocks are inside a `blocks` folder:
+
+```twig
+{% for block in post.meta( 'blocks' ) %}
+    {% 
+        include "blocks/#{ block.acf_fc_layout | sanitize ) }.twig"
+        with { block: block }
+    %}
+{% endfor %}
+```
+
+> The filter [sanitize](https://timber.github.io/docs/v2/guides/filters/#sanitize), slugify the block name
+
+And for example for a flexible content named `text` containing a `text` field, inside a `blocks/text.twig` file:
+
+```twig
+<p>{{ block.text }}</p>   
+```
```

Comment:
@19h47 I'm finding this example a little hard to understand. I think it might clarify if we use different names? for example:

A Flexible Content Field named `credit` containing a `text` field, inside a `blocks/photo.twig` file:

```twig
<p>Photo by {{ block.credit }}</p>   
```

... of course, I want to make sure that's correct and what you're actually trying to demonstrate

Reply:
Hello @jarednova, you totally right, my example with only the word `text` is weird and very redundant. A more real world example with a block named `photo` with a `credit` field is way better. 

Thank you for pointing this!